notebooks/0_prepare_transactions_dataset.ipynb

```diff
@@ -918,35 +919,8 @@
    },
    "outputs": [],
    "source": [
-    "transactions_df.to_csv(f'{data_dir}/transactions.csv', index=False)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "button": false,
-    "new_sheet": false,
-    "run_control": {
-     "read_only": false
-    }
-   },
-   "source": [
-    "#### Copy Local Data to S3 Bucket"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "button": false,
-    "new_sheet": false,
-    "run_control": {
-     "read_only": false
-    }
-   },
-   "outputs": [],
-   "source": [
-    "!aws s3 cp {data_dir}/transactions.csv s3://{BUCKET}/raw/transactions.csv"
+    "transactions_df.to_csv(f'{data_dir}/transactions.csv', index=False)\n",
+    "transactions_df.to_csv(f's3://{BUCKET}/raw/transactions.csv', index=False)"
```

Comment:
Any chance that this change has led to the error I received at bottom on notbook_0 ? 
Error received:
OSError: [Errno 12] Cannot allocate memory

Reply:
This change was necessary to avoid the `Cannot allocate memory` error now that the default notebook is a `ml.t3.medium`. Are you still seeing it with the change?

Reply:
As discussed, I had to switch to your new branch in order to get the modified notebook; it now works!